dodos/behavior.py

```diff
@@ -109,12 +109,16 @@ def datadiff_action(glob_pattern):
         if glob_pattern is not None:
             datadiff_args = datadiff_args + f"--glob-pattern '{glob_pattern}'"
 
-        return f"python3 -m behavior datadiff {datadiff_args}"
+        # Alter PYTHONPATH so it can find the cython compiled modules.
+        return f"PYTHONPATH=artifacts/behavior/plans:$PYTHONPATH python3 -m behavior datadiff {datadiff_args}"
 
     return {
         "actions": [
+            # This is necessary to force a rebuild everytime. Recompile diff_c.pyx.
+            "rm behavior/plans/diff_c.c",
```

Comment:
```suggestion
            "rm -f behavior/plans/diff_c.c",
```

Otherwise will fail when file doesn't exist.